homeassistant/components/trafikverket_ferry/coordinator.py

```diff
@@ -58,23 +58,23 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
         )
         self._from: str = entry.data[CONF_FROM]
         self._to: str = entry.data[CONF_TO]
-        self._time: time | None = parse_time(entry.data[CONF_TIME])
+        self._time: time | None = dt.parse_time(entry.data[CONF_TIME])
         self._weekdays: list[str] = entry.data[CONF_WEEKDAY]
 
     async def _async_update_data(self) -> dict[str, Any]:
         """Fetch data from Trafikverket."""
 
         departure_day = next_departuredate(self._weekdays)
-        currenttime = datetime.now(get_time_zone(self.hass.config.time_zone))
+        current_time = dt.now(dt.get_time_zone(self.hass.config.time_zone))
```

Comment:
We normally don't need to pass the timezone set in the config since the default timezone will match the config setting.

Reply:
Ok. That was not obvious on my very brief look so I will then remove the addition of the timezone here.